cirq/protocols/channel.py

```diff
@@ -175,3 +178,19 @@ def has_channel(val: Any) -> bool:
 
     # No has methods, use `_channel_` or delegates instead.
     return channel(val, None) is not None
+
+
+def validate_channel(supports_channel: SupportsChannel, rtol: int = 1e-5, atol: int = 1e-8):
+    """Validates that the channel's unitaries are a valid representation."""
```

Comment:
Channel's don't have unitaries :)  Doc should be something like "Validates that an object's channel Krauss operators are normalized."  

Also should also explicitly mention that this is
\sum_{k=0}^{r-1} A_i^\dagger A_i = I